{
  message: "Error occurred in service when creating user TypeError: Cannot read property 'create' of undefined",
  level: 'error',
  timestamp: '2021-12-01 22:00:34'
}
{
  message: "Error occurred in service when getting users TypeError: Cannot read property 'findAll' of undefined",
  level: 'error',
  timestamp: '2021-12-01 22:03:01'
}
{
  message: "Error encountered while getting users TypeError: Cannot read property 'findAll' of undefined",
  level: 'error',
  timestamp: '2021-12-01 22:03:01'
}
{
  message: 'Error occurred in service when getting users SequelizeDatabaseError: relation "users" does not exist',
  level: 'error',
  timestamp: '2021-12-01 22:17:17'
}
{
  message: 'Error encountered while getting users SequelizeDatabaseError: relation "users" does not exist',
  level: 'error',
  timestamp: '2021-12-01 22:17:17'
}
{
  message: 'Error occurred in service when creating user SequelizeDatabaseError: relation "users" does not exist',
  level: 'error',
  timestamp: '2021-12-01 22:17:37'
}
{
  message: "Error occurred in service when creating user TypeError: Cannot read property 'create' of undefined",
  level: 'error',
  timestamp: '2021-12-01 22:51:52'
}
{
  message: 'Error occurred in service when creating user SequelizeDatabaseError: relation "users" does not exist',
  level: 'error',
  timestamp: '2021-12-01 22:52:46'
}
{
  message: "Error occurred in service when creating user TypeError: Cannot read property 'create' of undefined",
  level: 'error',
  timestamp: '2021-12-01 23:15:06'
}
{
  message: 'Error occurred in service when creating user SequelizeDatabaseError: relation "users" does not exist',
  level: 'error',
  timestamp: '2021-12-01 23:31:00'
}
{
  message: "Error occurred in service when creating user TypeError: Cannot read property 'create' of undefined",
  level: 'error',
  timestamp: '2021-12-01 23:32:19'
}
{
  message: "Error occurred in service when creating user TypeError: Cannot read property 'create' of undefined",
  level: 'error',
  timestamp: '2021-12-01 23:39:28'
}
{
  message: "Error occurred in service when creating user TypeError: Cannot read property 'create' of undefined",
  level: 'error',
  timestamp: '2021-12-01 23:41:45'
}
{
  message: 'Error occurred in service when creating user [object Object]',
  level: 'error',
  timestamp: '2021-12-01 23:50:30'
}
{
  message: 'The database value[object Object]',
  level: 'error',
  timestamp: '2021-12-01 23:50:46'
}
{
  message: 'The database value[object Object]',
  level: 'error',
  timestamp: '2021-12-01 23:51:23'
}
{
  message: 'Error occurred in service when creating user SequelizeDatabaseError: relation "users" does not exist',
  level: 'error',
  timestamp: '2021-12-02 00:09:35'
}
{
  message: 'Error occurred in service when creating user SequelizeDatabaseError: column "id" of relation "users" does not exist',
  level: 'error',
  timestamp: '2021-12-02 00:15:08'
}
{
  message: 'Error encountered while getting users ReferenceError: allProperties is not defined',
  level: 'error',
  timestamp: '2021-12-02 00:23:50'
}
{
  message: 'Error occurred in service when getting users SequelizeDatabaseError: column "firstname" does not exist',
  level: 'error',
  timestamp: '2021-12-02 00:28:27'
}
{
  message: 'Error encountered while getting users SequelizeDatabaseError: column "firstname" does not exist',
  level: 'error',
  timestamp: '2021-12-02 00:28:27'
}
{
  message: 'Error getting user ReferenceError: id is not defined',
  level: 'error',
  timestamp: '2021-12-02 00:42:26'
}
{
  message: 'Error occurred in service when updating user details SequelizeDatabaseError: column "userId" does not exist',
  level: 'error',
  timestamp: '2021-12-02 00:43:35'
}
{
  message: 'Error when updating user details SequelizeDatabaseError: column "userId" does not exist',
  level: 'error',
  timestamp: '2021-12-02 00:43:35'
}
{
  message: 'Error occurred in service when updating user details ReferenceError: updateUser is not defined',
  level: 'error',
  timestamp: '2021-12-02 00:44:29'
}
{
  message: 'Error when updating user details ReferenceError: updateUser is not defined',
  level: 'error',
  timestamp: '2021-12-02 00:44:29'
}
{
  message: 'Error occurred in service when deleting user details ReferenceError: userid is not defined',
  level: 'error',
  timestamp: '2021-12-02 00:50:03'
}
{
  message: 'Error when deleting user details ReferenceError: userid is not defined',
  level: 'error',
  timestamp: '2021-12-02 00:50:03'
}
{
  message: 'Error occurred in service when deleting user details ReferenceError: userid is not defined',
  level: 'error',
  timestamp: '2021-12-02 00:50:07'
}
{
  message: 'Error when deleting user details ReferenceError: userid is not defined',
  level: 'error',
  timestamp: '2021-12-02 00:50:07'
}
{
  message: 'Error occurred in service when deleting user details TypeError: _index.default.destroy is not a function',
  level: 'error',
  timestamp: '2021-12-02 00:51:23'
}
{
  message: 'Error when deleting user details TypeError: _index.default.destroy is not a function',
  level: 'error',
  timestamp: '2021-12-02 00:51:23'
}
{
  message: 'Invalid user id {userId}',
  level: 'error',
  timestamp: '2021-12-04 12:15:48'
}
{
  message: 'Error when updating user details ReferenceError: createdUser is not defined',
  level: 'error',
  timestamp: '2021-12-04 12:36:47'
}
{
  message: 'Invalid urers id {id}',
  level: 'error',
  timestamp: '2021-12-04 12:41:29'
}
{
  message: 'Error getting user ReferenceError: id is not defined',
  level: 'error',
  timestamp: '2021-12-04 12:52:02'
}
{
  message: 'Invalid urers id {id}',
  level: 'error',
  timestamp: '2021-12-04 12:53:37'
}
{
  message: 'Invalid urers id {id}',
  level: 'error',
  timestamp: '2021-12-04 12:54:07'
}
{
  message: "Error when deleting user details TypeError: Cannot read property 'firstName' of undefined",
  level: 'error',
  timestamp: '2021-12-04 12:55:01'
}
{
  message: 'Error getting user ReferenceError: id is not defined',
  level: 'error',
  timestamp: '2021-12-04 12:55:19'
}
{
  message: 'Error getting user ReferenceError: id is not defined',
  level: 'error',
  timestamp: '2021-12-04 12:55:49'
}
{
  message: 'Error occurred in service when creating user SequelizeValidationError: notNull Violation: user.firstName cannot be null,\n' +
    'notNull Violation: user.lastName cannot be null,\n' +
    'notNull Violation: user.isAdmin cannot be null',
  level: 'error',
  timestamp: '2021-12-04 22:52:15'
}
{
  '0': 'm',
  '1': 'w',
  '2': 'a',
  '3': 's',
  '4': '1',
  '5': '@',
  '6': 'g',
  '7': 'm',
  '8': 'a',
  '9': 'i',
  '10': 'l',
  '11': '.',
  '12': 'c',
  '13': 'o',
  '14': 'm',
  level: 'error',
  message: 'No user found with this email',
  timestamp: '2021-12-04 23:31:34'
}
{
  '0': 'm',
  '1': 'w',
  '2': 'a',
  '3': 's',
  '4': '1',
  '5': '@',
  '6': 'g',
  '7': 'm',
  '8': 'a',
  '9': 'i',
  '10': 'l',
  '11': '.',
  '12': 'c',
  '13': 'o',
  '14': 'm',
  level: 'error',
  message: 'No user found with this email',
  timestamp: '2021-12-04 23:32:18'
}
{
  '0': 'm',
  '1': 'w',
  '2': 'a',
  '3': 's',
  '4': '1',
  '5': '@',
  '6': 'g',
  '7': 'm',
  '8': 'a',
  '9': 'i',
  '10': 'l',
  '11': '.',
  '12': 'c',
  '13': 'o',
  '14': 'm',
  level: 'error',
  message: 'No user found with this email',
  timestamp: '2021-12-04 23:32:48'
}
{
  '0': 'm',
  '1': 'w',
  '2': 'a',
  '3': 's',
  '4': '1',
  '5': '@',
  '6': 'g',
  '7': 'm',
  '8': 'a',
  '9': 'i',
  '10': 'l',
  '11': '.',
  '12': 'c',
  '13': 'o',
  '14': 'm',
  level: 'error',
  message: 'No user found with this email',
  timestamp: '2021-12-04 23:32:50'
}
{
  '0': 'm',
  '1': 'w',
  '2': 'a',
  '3': 's',
  '4': '1',
  '5': '@',
  '6': 'g',
  '7': 'm',
  '8': 'a',
  '9': 'i',
  '10': 'l',
  '11': '.',
  '12': 'c',
  '13': 'o',
  '14': 'm',
  level: 'error',
  message: 'No user found with this email',
  timestamp: '2021-12-04 23:33:22'
}
{
  message: '[object Object]',
  level: 'error',
  timestamp: '2021-12-05 13:08:16'
}
{
  message: '[object Object]',
  level: 'error',
  timestamp: '2021-12-05 13:12:00'
}
{
  message: '[object Object]',
  level: 'error',
  timestamp: '2021-12-05 13:19:19'
}
{
  message: '[object Object]',
  level: 'error',
  timestamp: '2021-12-05 13:20:43'
}
{
  message: '[object Object]',
  level: 'error',
  timestamp: '2021-12-05 13:21:47'
}
{
  message: '[object Object]',
  level: 'error',
  timestamp: '2021-12-05 13:37:48'
}
{
  message: 'Error occurred in service when getting users SequelizeDatabaseError: column "isAdmin" does not exist',
  level: 'error',
  timestamp: '2021-12-06 22:51:08'
}
{
  message: 'Error encountered while getting users SequelizeDatabaseError: column "isAdmin" does not exist',
  level: 'error',
  timestamp: '2021-12-06 22:51:08'
}
{
  '0': 'j',
  '1': 'a',
  '2': 'b',
  '3': 'a',
  '4': 'h',
  '5': '@',
  '6': 'g',
  '7': 'm',
  '8': 'a',
  '9': 'i',
  '10': 'l',
  '11': '.',
  '12': 'c',
  '13': 'o',
  '14': 'm',
  level: 'error',
  message: 'No user found with this email',
  timestamp: '2021-12-06 23:08:46'
}
{
  message: 'Error occurred in service when creating user SequelizeValidationError: notNull Violation: user.firstName cannot be null,\n' +
    'notNull Violation: user.lastName cannot be null',
  level: 'error',
  timestamp: '2021-12-06 23:09:53'
}
{
  level: 'warn',
  message: 'There was an error when trying to hash user password:  data and salt arguments required',
  stack: 'Error: data and salt arguments required\n' +
    '    at Object.hash (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\node_modules\\bcrypt\\bcrypt.js:137:17)\n' +
    '    at C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\api\\server\\controllers\\/UserController.js:30:16',
  timestamp: '2021-12-06 23:11:18'
}
{
  level: 'warn',
  message: 'There was an error when trying to hash user password:  data and salt arguments required',
  stack: 'Error: data and salt arguments required\n' +
    '    at Object.hash (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\node_modules\\bcrypt\\bcrypt.js:137:17)\n' +
    '    at C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\api\\server\\controllers\\/UserController.js:30:16',
  timestamp: '2021-12-06 23:11:35'
}
{
  message: 'Error occurred in service when adding contribution SequelizeValidationError: notNull Violation: contributions.amount cannot be null,\n' +
    'notNull Violation: contributions.dateDeposited cannot be null,\n' +
    'notNull Violation: contributions.amountPaid cannot be null,\n' +
    'notNull Violation: contributions.userId cannot be null',
  level: 'error',
  timestamp: '2021-12-14 23:57:55'
}
{
  message: 'Error occurred when adding user contributions SequelizeValidationError: notNull Violation: contributions.amount cannot be null,\n' +
    'notNull Violation: contributions.dateDeposited cannot be null,\n' +
    'notNull Violation: contributions.amountPaid cannot be null,\n' +
    'notNull Violation: contributions.userId cannot be null',
  level: 'error',
  timestamp: '2021-12-14 23:57:55'
}
{
  message: 'Error occurred in service when adding contribution SequelizeValidationError: notNull Violation: contributions.amount cannot be null,\n' +
    'notNull Violation: contributions.dateDeposited cannot be null,\n' +
    'notNull Violation: contributions.amountPaid cannot be null,\n' +
    'notNull Violation: contributions.userId cannot be null',
  level: 'error',
  timestamp: '2021-12-15 00:10:13'
}
{
  message: 'Error occurred when adding user contributions SequelizeValidationError: notNull Violation: contributions.amount cannot be null,\n' +
    'notNull Violation: contributions.dateDeposited cannot be null,\n' +
    'notNull Violation: contributions.amountPaid cannot be null,\n' +
    'notNull Violation: contributions.userId cannot be null',
  level: 'error',
  timestamp: '2021-12-15 00:10:13'
}
{
  message: 'Error occurred in service when adding contribution SequelizeValidationError: notNull Violation: contributions.amount cannot be null,\n' +
    'notNull Violation: contributions.dateDeposited cannot be null,\n' +
    'notNull Violation: contributions.amountPaid cannot be null,\n' +
    'notNull Violation: contributions.userId cannot be null',
  level: 'error',
  timestamp: '2021-12-15 00:10:15'
}
{
  message: 'Error occurred when adding user contributions SequelizeValidationError: notNull Violation: contributions.amount cannot be null,\n' +
    'notNull Violation: contributions.dateDeposited cannot be null,\n' +
    'notNull Violation: contributions.amountPaid cannot be null,\n' +
    'notNull Violation: contributions.userId cannot be null',
  level: 'error',
  timestamp: '2021-12-15 00:10:15'
}
{
  message: 'Error occurred in service when adding contribution SequelizeValidationError: notNull Violation: contributions.amount cannot be null,\n' +
    'notNull Violation: contributions.dateDeposited cannot be null,\n' +
    'notNull Violation: contributions.amountPaid cannot be null,\n' +
    'notNull Violation: contributions.userId cannot be null',
  level: 'error',
  timestamp: '2021-12-15 00:11:16'
}
{
  message: 'Error occurred when adding user contributions SequelizeValidationError: notNull Violation: contributions.amount cannot be null,\n' +
    'notNull Violation: contributions.dateDeposited cannot be null,\n' +
    'notNull Violation: contributions.amountPaid cannot be null,\n' +
    'notNull Violation: contributions.userId cannot be null',
  level: 'error',
  timestamp: '2021-12-15 00:11:16'
}
{
  message: 'Error occurred in service when adding contribution SequelizeValidationError: notNull Violation: contributions.amount cannot be null,\n' +
    'notNull Violation: contributions.dateDeposited cannot be null,\n' +
    'notNull Violation: contributions.amountPaid cannot be null,\n' +
    'notNull Violation: contributions.userId cannot be null',
  level: 'error',
  timestamp: '2021-12-15 00:21:27'
}
{
  message: 'Error occurred when adding user contributions SequelizeValidationError: notNull Violation: contributions.amount cannot be null,\n' +
    'notNull Violation: contributions.dateDeposited cannot be null,\n' +
    'notNull Violation: contributions.amountPaid cannot be null,\n' +
    'notNull Violation: contributions.userId cannot be null',
  level: 'error',
  timestamp: '2021-12-15 00:21:27'
}
{
  message: 'Error occurred in service when adding contribution SequelizeValidationError: notNull Violation: contributions.amount cannot be null,\n' +
    'notNull Violation: contributions.dateDeposited cannot be null,\n' +
    'notNull Violation: contributions.amountPaid cannot be null,\n' +
    'notNull Violation: contributions.userId cannot be null',
  level: 'error',
  timestamp: '2021-12-15 00:26:30'
}
{
  message: 'Error occurred when adding user contributions SequelizeValidationError: notNull Violation: contributions.amount cannot be null,\n' +
    'notNull Violation: contributions.dateDeposited cannot be null,\n' +
    'notNull Violation: contributions.amountPaid cannot be null,\n' +
    'notNull Violation: contributions.userId cannot be null',
  level: 'error',
  timestamp: '2021-12-15 00:26:30'
}
{
  message: 'Error occurred in service when adding contribution SequelizeValidationError: notNull Violation: contributions.amount cannot be null,\n' +
    'notNull Violation: contributions.dateDeposited cannot be null,\n' +
    'notNull Violation: contributions.amountPaid cannot be null,\n' +
    'notNull Violation: contributions.userId cannot be null',
  level: 'error',
  timestamp: '2021-12-15 22:48:30'
}
{
  message: 'Error occurred when adding user contributions SequelizeValidationError: notNull Violation: contributions.amount cannot be null,\n' +
    'notNull Violation: contributions.dateDeposited cannot be null,\n' +
    'notNull Violation: contributions.amountPaid cannot be null,\n' +
    'notNull Violation: contributions.userId cannot be null',
  level: 'error',
  timestamp: '2021-12-15 22:48:30'
}
{
  message: 'Error occurred in service when adding contribution SequelizeValidationError: notNull Violation: contributions.amount cannot be null,\n' +
    'notNull Violation: contributions.dateDeposited cannot be null,\n' +
    'notNull Violation: contributions.amountPaid cannot be null,\n' +
    'notNull Violation: contributions.userId cannot be null',
  level: 'error',
  timestamp: '2021-12-20 13:26:32'
}
{
  message: 'Error occurred when adding user contributions SequelizeValidationError: notNull Violation: contributions.amount cannot be null,\n' +
    'notNull Violation: contributions.dateDeposited cannot be null,\n' +
    'notNull Violation: contributions.amountPaid cannot be null,\n' +
    'notNull Violation: contributions.userId cannot be null',
  level: 'error',
  timestamp: '2021-12-20 13:26:32'
}
{
  message: 'Error occurred in service when adding contribution SequelizeValidationError: notNull Violation: contributions.amount cannot be null,\n' +
    'notNull Violation: contributions.dateDeposited cannot be null,\n' +
    'notNull Violation: contributions.amountPaid cannot be null,\n' +
    'notNull Violation: contributions.userId cannot be null',
  level: 'error',
  timestamp: '2021-12-20 14:14:01'
}
{
  message: 'Error occurred when adding user contributions SequelizeValidationError: notNull Violation: contributions.amount cannot be null,\n' +
    'notNull Violation: contributions.dateDeposited cannot be null,\n' +
    'notNull Violation: contributions.amountPaid cannot be null,\n' +
    'notNull Violation: contributions.userId cannot be null',
  level: 'error',
  timestamp: '2021-12-20 14:14:01'
}
{
  message: 'Error occurred in service when adding contribution SequelizeValidationError: notNull Violation: contributions.amount cannot be null,\n' +
    'notNull Violation: contributions.dateDeposited cannot be null,\n' +
    'notNull Violation: contributions.amountPaid cannot be null,\n' +
    'notNull Violation: contributions.userId cannot be null',
  level: 'error',
  timestamp: '2021-12-20 14:16:33'
}
{
  message: 'Error occurred when adding user contributions SequelizeValidationError: notNull Violation: contributions.amount cannot be null,\n' +
    'notNull Violation: contributions.dateDeposited cannot be null,\n' +
    'notNull Violation: contributions.amountPaid cannot be null,\n' +
    'notNull Violation: contributions.userId cannot be null',
  level: 'error',
  timestamp: '2021-12-20 14:16:33'
}
{
  message: 'Error occurred in service when adding contribution SequelizeDatabaseError: column "balance" does not exist',
  level: 'error',
  timestamp: '2021-12-20 15:30:09'
}
{
  message: 'Error occurred when adding user contributions SequelizeDatabaseError: column "balance" does not exist',
  level: 'error',
  timestamp: '2021-12-20 15:30:09'
}
{
  message: 'Error occurred in service when adding contribution SequelizeDatabaseError: column "balance" does not exist',
  level: 'error',
  timestamp: '2021-12-20 15:33:35'
}
{
  message: 'Error occurred when adding user contributions SequelizeDatabaseError: column "balance" does not exist',
  level: 'error',
  timestamp: '2021-12-20 15:33:35'
}
{
  message: 'Error occurred in service when adding contribution SequelizeDatabaseError: column "balance" does not exist',
  level: 'error',
  timestamp: '2021-12-21 10:46:20'
}
{
  message: 'Error occurred when adding user contributions SequelizeDatabaseError: column "balance" does not exist',
  level: 'error',
  timestamp: '2021-12-21 10:46:20'
}
{
  '0': 's',
  '1': 't',
  '2': '@',
  '3': 'g',
  '4': 'm',
  '5': 'a',
  '6': 'i',
  '7': 'l',
  '8': '.',
  '9': 'c',
  '10': 'o',
  '11': 'm',
  level: 'error',
  message: 'No user found with this email',
  timestamp: '2021-12-21 11:33:49'
}
{
  message: 'Error occurred in service when adding contribution SequelizeValidationError: notNull Violation: contributions.userId cannot be null',
  level: 'error',
  timestamp: '2021-12-21 11:45:36'
}
{
  message: 'Error occurred when adding user contributions SequelizeValidationError: notNull Violation: contributions.userId cannot be null',
  level: 'error',
  timestamp: '2021-12-21 11:45:36'
}
{
  message: 'Error getting user contributions ReferenceError: error is not defined',
  level: 'error',
  timestamp: '2021-12-21 13:52:00'
}
{
  message: 'Error occurred in service when fetching individual user contributions TypeError: _index.default.contributions.get is not a function',
  level: 'error',
  timestamp: '2021-12-21 13:52:57'
}
{
  message: 'Error getting user contributions TypeError: _index.default.contributions.get is not a function',
  level: 'error',
  timestamp: '2021-12-21 13:52:57'
}
{
  message: 'Invalid user id uiuu',
  level: 'error',
  timestamp: '2021-12-21 14:09:02'
}
{
  message: 'Invalid user id uiuu',
  level: 'error',
  timestamp: '2021-12-21 14:09:49'
}
{
  message: 'Invalid user id uiuu',
  level: 'error',
  timestamp: '2021-12-21 14:09:52'
}
{
  message: 'Invalid user id uiuu',
  level: 'error',
  timestamp: '2021-12-21 14:10:16'
}
{
  message: 'Invalid user id ""',
  level: 'error',
  timestamp: '2021-12-21 14:10:31'
}
{
  message: 'Error getting user contributions ReferenceError: id is not defined',
  level: 'error',
  timestamp: '2021-12-21 14:11:41'
}
{
  message: 'Error getting user contributions ReferenceError: userid is not defined',
  level: 'error',
  timestamp: '2021-12-21 14:12:20'
}
{
  message: 'Error getting user contributions ReferenceError: userid is not defined',
  level: 'error',
  timestamp: '2021-12-21 14:12:22'
}
{
  message: 'Invalid user id 89',
  level: 'error',
  timestamp: '2021-12-21 14:13:11'
}
{
  message: 'Invalid user id 89',
  level: 'error',
  timestamp: '2021-12-21 14:13:42'
}
{
  message: 'Error occurred in service when retrieving all contribution TypeError: _index.default.findAll is not a function',
  level: 'error',
  timestamp: '2021-12-22 15:24:48'
}
{
  message: 'Error encountered while getting contributions TypeError: _index.default.findAll is not a function',
  level: 'error',
  timestamp: '2021-12-22 15:24:48'
}
{
  message: 'Error encountered while getting contributions ReferenceError: res is not defined',
  level: 'error',
  timestamp: '2021-12-22 15:26:41'
}
{
  code: 'EADDRINUSE',
  errno: -4091,
  syscall: 'listen',
  address: '::',
  port: 8000,
  level: 'error',
  message: 'There was an error with the app at port 8000:  listen EADDRINUSE: address already in use :::8000',
  stack: 'Error: listen EADDRINUSE: address already in use :::8000\n' +
    '    at Server.setupListenHandle [as _listen2] (net.js:1331:16)\n' +
    '    at listenInCluster (net.js:1379:12)\n' +
    '    at Server.listen (net.js:1465:7)\n' +
    '    at Function.listen (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\node_modules\\express\\lib\\application.js:618:24)\n' +
    '    at Object.<anonymous> (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\api\\/index.js:35:4)\n' +
    '    at Module._compile (internal/modules/cjs/loader.js:1085:14)\n' +
    '    at Module._compile (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\node_modules\\pirates\\lib\\index.js:99:24)\n' +
    '    at Module._extensions..js (internal/modules/cjs/loader.js:1114:10)\n' +
    '    at Object.newLoader [as .js] (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\node_modules\\pirates\\lib\\index.js:104:7)\n' +
    '    at Module.load (internal/modules/cjs/loader.js:950:32)',
  timestamp: '2021-12-22 17:38:46'
}
{
  code: 'EADDRINUSE',
  errno: -4091,
  syscall: 'listen',
  address: '::',
  port: 8000,
  level: 'error',
  message: 'There was an error with the app at port 8000:  listen EADDRINUSE: address already in use :::8000',
  stack: 'Error: listen EADDRINUSE: address already in use :::8000\n' +
    '    at Server.setupListenHandle [as _listen2] (net.js:1331:16)\n' +
    '    at listenInCluster (net.js:1379:12)\n' +
    '    at Server.listen (net.js:1465:7)\n' +
    '    at Function.listen (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\node_modules\\express\\lib\\application.js:618:24)\n' +
    '    at Object.<anonymous> (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\api\\/index.js:35:4)\n' +
    '    at Module._compile (internal/modules/cjs/loader.js:1085:14)\n' +
    '    at Module._compile (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\node_modules\\pirates\\lib\\index.js:99:24)\n' +
    '    at Module._extensions..js (internal/modules/cjs/loader.js:1114:10)\n' +
    '    at Object.newLoader [as .js] (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\node_modules\\pirates\\lib\\index.js:104:7)\n' +
    '    at Module.load (internal/modules/cjs/loader.js:950:32)',
  timestamp: '2021-12-22 17:39:16'
}
{
  code: 'EADDRINUSE',
  errno: -4091,
  syscall: 'listen',
  address: '::',
  port: 8000,
  level: 'error',
  message: 'There was an error with the app at port 8000:  listen EADDRINUSE: address already in use :::8000',
  stack: 'Error: listen EADDRINUSE: address already in use :::8000\n' +
    '    at Server.setupListenHandle [as _listen2] (net.js:1331:16)\n' +
    '    at listenInCluster (net.js:1379:12)\n' +
    '    at Server.listen (net.js:1465:7)\n' +
    '    at Function.listen (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\node_modules\\express\\lib\\application.js:618:24)\n' +
    '    at Object.<anonymous> (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\api\\/index.js:35:4)\n' +
    '    at Module._compile (internal/modules/cjs/loader.js:1085:14)\n' +
    '    at Module._compile (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\node_modules\\pirates\\lib\\index.js:99:24)\n' +
    '    at Module._extensions..js (internal/modules/cjs/loader.js:1114:10)\n' +
    '    at Object.newLoader [as .js] (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\node_modules\\pirates\\lib\\index.js:104:7)\n' +
    '    at Module.load (internal/modules/cjs/loader.js:950:32)',
  timestamp: '2021-12-22 17:39:27'
}
{
  code: 'EADDRINUSE',
  errno: -4091,
  syscall: 'listen',
  address: '::',
  port: 8000,
  level: 'error',
  message: 'There was an error with the app at port 8000:  listen EADDRINUSE: address already in use :::8000',
  stack: 'Error: listen EADDRINUSE: address already in use :::8000\n' +
    '    at Server.setupListenHandle [as _listen2] (net.js:1331:16)\n' +
    '    at listenInCluster (net.js:1379:12)\n' +
    '    at Server.listen (net.js:1465:7)\n' +
    '    at Function.listen (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\node_modules\\express\\lib\\application.js:618:24)\n' +
    '    at Object.<anonymous> (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\api\\/index.js:35:4)\n' +
    '    at Module._compile (internal/modules/cjs/loader.js:1085:14)\n' +
    '    at Module._compile (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\node_modules\\pirates\\lib\\index.js:99:24)\n' +
    '    at Module._extensions..js (internal/modules/cjs/loader.js:1114:10)\n' +
    '    at Object.newLoader [as .js] (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\node_modules\\pirates\\lib\\index.js:104:7)\n' +
    '    at Module.load (internal/modules/cjs/loader.js:950:32)',
  timestamp: '2021-12-22 18:14:18'
}
{
  code: 'EADDRINUSE',
  errno: -4091,
  syscall: 'listen',
  address: '::',
  port: 8000,
  level: 'error',
  message: 'There was an error with the app at port 8000:  listen EADDRINUSE: address already in use :::8000',
  stack: 'Error: listen EADDRINUSE: address already in use :::8000\n' +
    '    at Server.setupListenHandle [as _listen2] (net.js:1331:16)\n' +
    '    at listenInCluster (net.js:1379:12)\n' +
    '    at Server.listen (net.js:1465:7)\n' +
    '    at Function.listen (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\node_modules\\express\\lib\\application.js:618:24)\n' +
    '    at Object.<anonymous> (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\api\\/index.js:35:4)\n' +
    '    at Module._compile (internal/modules/cjs/loader.js:1085:14)\n' +
    '    at Module._compile (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\node_modules\\pirates\\lib\\index.js:99:24)\n' +
    '    at Module._extensions..js (internal/modules/cjs/loader.js:1114:10)\n' +
    '    at Object.newLoader [as .js] (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\node_modules\\pirates\\lib\\index.js:104:7)\n' +
    '    at Module.load (internal/modules/cjs/loader.js:950:32)',
  timestamp: '2021-12-22 18:15:18'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-22 18:30:10'
}
{
  message: 'Error occurred in service when retrieving all contribution ReferenceError: User is not defined',
  level: 'error',
  timestamp: '2021-12-22 18:30:23'
}
{
  message: 'Error encountered while getting contributions ReferenceError: User is not defined',
  level: 'error',
  timestamp: '2021-12-22 18:30:23'
}
{
  message: 'Error occurred in service when retrieving all contribution ReferenceError: user is not defined',
  level: 'error',
  timestamp: '2021-12-22 18:31:36'
}
{
  message: 'Error encountered while getting contributions ReferenceError: user is not defined',
  level: 'error',
  timestamp: '2021-12-22 18:31:36'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-22 18:32:27'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-22 18:32:38'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-22 18:36:43'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-22 18:37:21'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-22 18:38:11'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-22 18:41:29'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-22 18:43:32'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-22 18:44:12'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-22 18:44:54'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-22 18:47:14'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-23 11:27:22'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-23 11:27:24'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-23 11:27:26'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-23 11:32:14'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-23 11:40:19'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-23 11:41:23'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-23 11:50:44'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-23 11:51:48'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-23 12:06:27'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-23 12:07:29'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-23 12:07:57'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-23 12:09:03'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-23 12:09:38'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-23 12:19:18'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-23 12:19:20'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-23 12:19:43'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-23 12:20:37'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-23 12:20:39'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-23 12:23:12'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-23 12:23:48'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-23 12:23:50'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2021-12-23 15:16:19'
}
{
  message: 'Error occurred in service when fetching user details SequelizeDatabaseError: column "nan" does not exist',
  level: 'error',
  timestamp: '2022-01-08 10:10:45'
}
{
  message: 'Error occurred in service when fetching user details SequelizeDatabaseError: column "nan" does not exist',
  level: 'error',
  timestamp: '2022-01-08 10:13:03'
}
{
  message: "Error occurred in service when fetching data to login user Error: Invalid value { email: 'kimari@gmail.com' }",
  level: 'error',
  timestamp: '2022-01-08 10:18:07'
}
{
  '0': 'm',
  '1': 'w',
  '2': 'a',
  '3': 's',
  '4': '1',
  '5': '1',
  '6': '@',
  '7': 'g',
  '8': 'm',
  '9': 'a',
  '10': 'i',
  '11': 'l',
  '12': '.',
  '13': 'c',
  '14': 'o',
  '15': 'm',
  level: 'error',
  message: 'No user found with this email',
  timestamp: '2022-01-08 10:31:27'
}
{
  '0': 'm',
  '1': 'w',
  '2': 'a',
  '3': 's',
  '4': '1',
  '5': '1',
  '6': '@',
  '7': 'g',
  '8': 'm',
  '9': 'a',
  '10': 'i',
  '11': 'l',
  '12': '.',
  '13': 'c',
  '14': 'o',
  '15': 'm',
  level: 'error',
  message: 'No user found with this email',
  timestamp: '2022-01-08 10:31:38'
}
{
  message: 'No contribution found under the provided id',
  level: 'error',
  timestamp: '2022-01-08 10:34:14'
}
{
  '0': 'm',
  '1': 'w',
  '2': 'a',
  '3': 's',
  '4': '1',
  '5': '@',
  '6': 'g',
  '7': 'm',
  '8': 'a',
  '9': 'i',
  '10': 'l',
  '11': '.',
  '12': 'c',
  '13': 'o',
  '14': 'm',
  level: 'error',
  message: 'No user found with this email',
  timestamp: '2022-02-08 08:55:15'
}
{
  '0': 'm',
  '1': 'w',
  '2': 'a',
  '3': 'n',
  '4': 'g',
  '5': 'i',
  '6': 'f',
  '7': '1',
  '8': '8',
  '9': '4',
  '10': '@',
  '11': 'g',
  '12': 'm',
  '13': 'a',
  '14': 'i',
  '15': 'l',
  '16': '.',
  '17': 'c',
  '18': 'o',
  '19': 'm',
  level: 'error',
  message: 'No user found with this email',
  timestamp: '2022-02-09 17:33:52'
}
{
  '0': 'm',
  '1': 'w',
  '2': 'a',
  '3': 'n',
  '4': 'g',
  '5': 'i',
  '6': 'f',
  '7': '1',
  '8': '8',
  '9': '4',
  '10': '@',
  '11': 'g',
  '12': 'm',
  '13': 'a',
  '14': 'i',
  '15': 'l',
  '16': '.',
  '17': 'c',
  '18': 'o',
  '19': 'm',
  level: 'error',
  message: 'No user found with this email',
  timestamp: '2022-02-09 17:33:56'
}
{
  '0': 'm',
  '1': 'w',
  '2': 'a',
  '3': 'n',
  '4': 'g',
  '5': 'i',
  '6': 'f',
  '7': '1',
  '8': '8',
  '9': '4',
  '10': '@',
  '11': 'g',
  '12': 'm',
  '13': 'a',
  '14': 'i',
  '15': 'l',
  '16': '.',
  '17': 'c',
  '18': 'o',
  '19': 'm',
  level: 'error',
  message: 'No user found with this email',
  timestamp: '2022-02-11 08:42:39'
}
{
  code: 'EADDRINUSE',
  errno: -4091,
  syscall: 'listen',
  address: '::',
  port: 8000,
  level: 'error',
  message: 'There was an error with the app at port 8000:  listen EADDRINUSE: address already in use :::8000',
  stack: 'Error: listen EADDRINUSE: address already in use :::8000\n' +
    '    at Server.setupListenHandle [as _listen2] (net.js:1331:16)\n' +
    '    at listenInCluster (net.js:1379:12)\n' +
    '    at Server.listen (net.js:1465:7)\n' +
    '    at Function.listen (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\node_modules\\express\\lib\\application.js:618:24)\n' +
    '    at Object.<anonymous> (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\api\\/index.js:49:4)\n' +
    '    at Module._compile (internal/modules/cjs/loader.js:1085:14)\n' +
    '    at Module._compile (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\node_modules\\pirates\\lib\\index.js:99:24)\n' +
    '    at Module._extensions..js (internal/modules/cjs/loader.js:1114:10)\n' +
    '    at Object.newLoader [as .js] (C:\\Users\\Francis\\Desktop\\mwas\\Jamhurican-backend\\node_modules\\pirates\\lib\\index.js:104:7)\n' +
    '    at Module.load (internal/modules/cjs/loader.js:950:32)',
  timestamp: '2022-02-16 11:46:55'
}
{
  '0': 'j',
  '1': 'e',
  '2': 's',
  '3': 's',
  '4': 'e',
  '5': '.',
  '6': 'g',
  '7': 'i',
  '8': 't',
  '9': 'o',
  '10': 'n',
  '11': 'g',
  '12': 'a',
  '13': '@',
  '14': 'g',
  '15': 'm',
  '16': 'a',
  '17': 'i',
  '18': 'l',
  '19': '.',
  '20': 'c',
  '21': 'o',
  '22': 'm',
  level: 'error',
  message: 'No user found with this email',
  timestamp: '2022-02-19 14:25:21'
}
{
  '0': 'j',
  '1': 'e',
  '2': 's',
  '3': 's',
  '4': 'e',
  '5': '.',
  '6': 'g',
  '7': 'i',
  '8': 't',
  '9': 'o',
  '10': 'n',
  '11': 'g',
  '12': 'a',
  '13': '@',
  '14': 'g',
  '15': 'm',
  '16': 'a',
  '17': 'i',
  '18': 'l',
  '19': '.',
  '20': 'c',
  '21': 'o',
  '22': 'm',
  level: 'error',
  message: 'No user found with this email',
  timestamp: '2022-02-19 14:29:01'
}
{
  '0': 'm',
  '1': 'w',
  '2': 'a',
  '3': 'n',
  '4': 'g',
  '5': 'i',
  '6': 'f',
  '7': 'r',
  '8': 'a',
  '9': 'n',
  '10': 'c',
  '11': 'i',
  '12': 's',
  '13': '8',
  '14': '5',
  '15': '9',
  '16': '@',
  '17': 'g',
  '18': 'm',
  '19': 'a',
  '20': 'i',
  '21': 'l',
  '22': '.',
  '23': 'c',
  '24': 'o',
  '25': 'm',
  level: 'error',
  message: 'No user found with this email',
  timestamp: '2022-02-19 14:32:35'
}
{
  '0': 'm',
  '1': 'w',
  '2': 'a',
  '3': 'n',
  '4': 'g',
  '5': 'i',
  '6': 'f',
  '7': 'r',
  '8': 'a',
  '9': 'n',
  '10': 'c',
  '11': 'i',
  '12': 's',
  '13': '8',
  '14': '5',
  '15': '9',
  '16': '@',
  '17': 'g',
  '18': 'm',
  '19': 'a',
  '20': 'i',
  '21': 'l',
  '22': '.',
  '23': 'c',
  '24': 'o',
  '25': 'm',
  level: 'error',
  message: 'No user found with this email',
  timestamp: '2022-02-19 14:35:12'
}
{
  message: 'Invalid user id undefined',
  level: 'error',
  timestamp: '2022-02-20 20:54:28'
}
{
  message: 'Invalid user id undefined',
  level: 'error',
  timestamp: '2022-02-20 20:54:36'
}
{
  message: 'Invalid user id undefined',
  level: 'error',
  timestamp: '2022-02-20 20:54:47'
}
{
  message: 'Invalid user id undefined',
  level: 'error',
  timestamp: '2022-02-21 08:47:54'
}
{
  message: 'Invalid user id undefined',
  level: 'error',
  timestamp: '2022-02-21 08:47:57'
}
{
  message: 'Invalid user id undefined',
  level: 'error',
  timestamp: '2022-02-21 08:48:00'
}
{
  message: 'Invalid user id undefined',
  level: 'error',
  timestamp: '2022-02-21 08:48:01'
}
{
  message: 'Invalid user id undefined',
  level: 'error',
  timestamp: '2022-02-21 08:48:33'
}
{
  message: 'Invalid user id undefined',
  level: 'error',
  timestamp: '2022-02-21 08:48:40'
}
{
  message: 'Invalid user id undefined',
  level: 'error',
  timestamp: '2022-02-21 08:48:44'
}
{
  message: 'Invalid user id undefined',
  level: 'error',
  timestamp: '2022-02-21 08:49:16'
}
{
  message: 'Invalid user id undefined',
  level: 'error',
  timestamp: '2022-02-21 08:49:16'
}
{
  message: 'Invalid user id undefined',
  level: 'error',
  timestamp: '2022-02-21 08:49:35'
}
{
  message: 'Invalid user id undefined',
  level: 'error',
  timestamp: '2022-02-21 08:49:42'
}
{
  message: 'Invalid user id undefined',
  level: 'error',
  timestamp: '2022-02-21 08:52:37'
}
{
  message: 'Invalid user id undefined',
  level: 'error',
  timestamp: '2022-02-21 08:52:44'
}
{
  message: 'Invalid user id undefined',
  level: 'error',
  timestamp: '2022-02-21 08:52:47'
}
{
  message: 'Invalid user id undefined',
  level: 'error',
  timestamp: '2022-02-21 08:54:04'
}
{
  message: 'Invalid user id NaN',
  level: 'error',
  timestamp: '2022-02-21 09:45:57'
}
{
  message: 'Invalid user id NaN',
  level: 'error',
  timestamp: '2022-02-21 09:51:46'
}
{
  message: 'Invalid user id NaN',
  level: 'error',
  timestamp: '2022-02-21 09:53:33'
}
{
  message: 'Invalid user id NaN',
  level: 'error',
  timestamp: '2022-02-21 11:24:04'
}
{
  message: 'Invalid user id NaN',
  level: 'error',
  timestamp: '2022-02-21 11:24:08'
}
{
  message: 'Invalid user id NaN',
  level: 'error',
  timestamp: '2022-02-21 11:32:17'
}
{
  message: 'Invalid user id NaN',
  level: 'error',
  timestamp: '2022-02-21 11:33:18'
}
{
  message: 'Invalid user id NaN',
  level: 'error',
  timestamp: '2022-02-21 11:45:46'
}
{
  message: 'Error occurred in service when adding contribution SequelizeDatabaseError: invalid input syntax for type timestamp with time zone: "Invalid date"',
  level: 'error',
  timestamp: '2022-02-23 22:25:21'
}
{
  message: 'Error occurred when adding user contributions SequelizeDatabaseError: invalid input syntax for type timestamp with time zone: "Invalid date"',
  level: 'error',
  timestamp: '2022-02-23 22:25:21'
}
{
  '0': 'm',
  '1': 'w',
  '2': 'a',
  '3': 'n',
  '4': 'g',
  '5': 'i',
  '6': 'f',
  '7': '1',
  '8': '8',
  '9': '4',
  '10': '@',
  '11': 'g',
  '12': 'm',
  '13': 'a',
  '14': 'i',
  '15': 'l',
  '16': '.',
  '17': 'c',
  '18': 'o',
  '19': 'm',
  level: 'error',
  message: 'No user found with this email',
  timestamp: '2022-02-24 08:57:40'
}
{
  message: 'Invalid user id NaN',
  level: 'error',
  timestamp: '2022-02-24 09:36:27'
}
{
  message: 'Invalid user id NaN',
  level: 'error',
  timestamp: '2022-02-24 10:16:40'
}
{
  message: 'Error getting user ReferenceError: id is not defined',
  level: 'error',
  timestamp: '2022-02-24 11:48:06'
}
{
  message: 'Error getting user ReferenceError: id is not defined',
  level: 'error',
  timestamp: '2022-02-24 11:54:42'
}
{
  message: 'Error getting user ReferenceError: id is not defined',
  level: 'error',
  timestamp: '2022-02-24 11:55:06'
}
{
  message: 'Error getting user ReferenceError: id is not defined',
  level: 'error',
  timestamp: '2022-02-24 11:57:35'
}
{
  message: 'Error getting user ReferenceError: id is not defined',
  level: 'error',
  timestamp: '2022-02-24 11:58:51'
}
{
  message: 'Error getting user ReferenceError: id is not defined',
  level: 'error',
  timestamp: '2022-02-24 12:00:20'
}
{
  message: 'Error getting user ReferenceError: id is not defined',
  level: 'error',
  timestamp: '2022-02-24 12:00:42'
}
{
  message: 'Error getting user ReferenceError: id is not defined',
  level: 'error',
  timestamp: '2022-02-24 12:03:13'
}
{
  message: 'Error getting user ReferenceError: id is not defined',
  level: 'error',
  timestamp: '2022-02-24 12:06:22'
}
{
  message: 'Error getting user ReferenceError: id is not defined',
  level: 'error',
  timestamp: '2022-02-24 12:06:45'
}
